activation/activation_test.go

```diff
@@ -287,8 +287,7 @@ func TestBuilder_waitForFirstATX(t *testing.T) {
 
 func TestBuilder_StartSmeshingCoinbase(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
-	// cancel the parent context, so it doesn't run the building loop
-	cancel()
+	defer cancel()
 	tab := newTestBuilder(t, WithContext(ctx))
```

Comment:
I think you can just use `context.Background()` here instead of `ctx`. `ctx` is only canceled when the test finishes so there isn't really a use to it?

Reply:
Removed the custom context altogether (the default is to use the background one).